lib/util/git/private-key.spec.ts

```diff
@@ -10,39 +12,121 @@ jest.mock('fs-extra', () =>
     >('../../../test/fixtures')
     .fsExtra(),
 );
-jest.mock('../exec');
+jest.mock('../exec', () =>
+  jest
+    .requireActual<typeof import('jest-mock-extended')>('jest-mock-extended')
+    .mock<MockProxy<typeof execModule>>(),
+);
 
-const exec = mocked(exec_);
+const exec = (execModule as MockProxy<typeof execModule>).exec;
```

Comment:
See https://github.com/marchaos/jest-mock-extended/issues/87#issuecomment-1419801660 for the source of inspiration.